JASP-Engine/JASP/R/descriptives.R

```diff
@@ -612,19 +597,29 @@ Descriptives <- function(jaspResults, dataset, options) {
 
   # first do the diagonal and store breaks
   for (row in seq_along(variables)) {
-  	plotMat[[row, row]] <- .plotMarginalCorDescriptives(dataset[[variables[[row]]]]) + adjMargin
-  	axisBreaks[[row]] <- JASPgraphs::getAxisBreaks(plotMat[[row, row]])
+    if (variable.statuses[[row]]$error != "") {
+      plotMat[[row, row]] <- .displayError(errorMessage=variable.statuses[[row]]$error)
+    } else {
+    	plotMat[[row, row]] <- .plotMarginalCorDescriptives(dataset[[.v(variables[[row]])]]) + adjMargin
+    	axisBreaks[[row]] <- JASPgraphs::getAxisBreaks(plotMat[[row, row]])
+    }
   }
 
   # now do off-diagonal and use the same breaks
   for (row in seq_len(l-1)) {
   	for (col in seq(row+1, l)) {
-  		plotMat[[row, col]] <- .plotScatterDescriptives(
-  			xVar    = dataset[[variables[[col]]]],
-  			yVar    = dataset[[variables[[row]]]],
-  			xBreaks = axisBreaks[[col]]$x,
-  			yBreaks = axisBreaks[[row]]$x
-  		) + adjMargin
+	    if (variable.statuses[[row]]$error != "") {
```

Comment:
Mixing tabs and spaces eh?

Reply:
In my defence the tabs were already there.. I'll make everything spaces though

Reply:
I know you didn't cause it but it's one of the easy things to improve :)